hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java

```diff
@@ -101,7 +101,7 @@
   }
 
   public static void initTestSetup() {
-    Dispatcher.setBlockMoveWaitTime(1000L) ;
+//    Dispatcher.setBlockMoveWaitTime(1000L) ;
```

Comment:
Can you remove the code instead of commenting it out?

Reply:
fxd